index.bs

```diff
@@ -312,8 +330,13 @@ Whenever a <a>significant change in orientation</a> occurs, the user agent must
 
   1. If the implementation cannot provide <a>relative orientation</a> or the resulting <a>absolute orientation</a> data is more accurate:
     1. Let <var>absolute</var> be true.
+    1. Let <var>policies</var> be « "<a data-lt="accelerometer-feature"><code>accelerometer</code></a>", "<a data-lt="gyroscope-feature"><code>gyroscope</code></a>", "<a data-lt="magnetometer-feature"><code>magnetometer</code></a>" ».
   1. Otherwise:
     1. Let <var>absolute</var> be false.
+    1. Let <var>policies</var> be « "<a data-lt="accelerometer-feature"><code>accelerometer</code></a>", "<a data-lt="gyroscope-feature"><code>gyroscope</code></a>" ».
+  1. Let <var>document</var> be <var>window</var>'s <a>associated Document</a>.
+  1. <a for="list">For each</a> <var>policy</var> of <var>policies</var>:
+      1. If <var>document</var> is not <a>allowed to use</a> the <a>policy-controlled feature</a> named <var>policy</var>, return.
```

Comment:
There an open question here for both the permissions and permissions policy integrations about whether a failure means that an all-null event is fired. I don't recall what Chromium does.

Reply:
requestPermission() just resolves with the current permission at the moment, but the permissions policy failures do not fire any event AFAICS: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/device_orientation/device_motion_controller.cc;l=70;drc=047c7dc4ee1ce908d7fea38ca063fa2f80f92c77

WebKit seems to do the same: https://github.com/WebKit/WebKit/blob/8dcff579b57ecdd441f9e2b54a54299a57c048ed/Source/WebCore/page/LocalDOMWindow.cpp#L2171